src/test/groovy/org/ods/doc/gen/core/test/wiremock/WiremockManager.groovy

```diff
@@ -34,6 +34,9 @@ class WiremockManager {
 
     WiremockManager withScenario(String scenario){
```

Comment:
I don't see why call withScenario method... without scenario

Reply:
Removed it.